src/main/kotlin/com/github/suusan2go/kotlinfillclass/inspections/ValueGenerator.kt

```diff
@@ -0,0 +1,36 @@
+package com.github.suusan2go.kotlinfillclass.inspections
+
+import com.thedeanda.lorem.LoremIpsum
+
+object ValueGenerator {
```

Comment:
If possible, could you please add a unit test for this class as well?

Reply:
@oboenikui Sure, got busy at work. Will try to do that in the weekend

Reply:
Since the test was not expected to be added for a while, I created it on my end. I have also changed some implementations to improve functionality.

https://github.com/suusan2go/kotlin-fill-class/commit/8b9dfdd8c352fd62591e8ae6a9a339552e60dfb0